### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Aikhjarto** [chriskiehl/Gooey#329]
It would be nice to support an integer type which defaults to None. Would be something like the following possible with gooey?
```
parser.add_argument('--start-index', type=int, default=None)
```

This is supported by argparse but gooey reports the following error:
```
  File "(...longpath...)\site-packages\gooey\gui\components\widgets\dropdown.py", line 29, in setValue
    index = self._meta['choices'].index(value) + 1
ValueError: 'None' is not in list
```

**Comment by chriskiehl** [comment]
Please fill out the issue template.

**Comment by Aikhjarto** [comment]
### Appropriate fields from the issue template
 - OS: Windows 7
 - Python Version: 3.6
 - Gooey Version: 1.0.2-release branch
 - Expected Behavior: using ```type=int``` should work with ```default=None```
 - Actual Behavior: crash with ```ValueError: 'None' is not in list```
### Minimal crashing example
```python
import gooey

@gooey.Gooey
def main():
    parser=gooey.GooeyParser()
    parser.add_argument('--test', default=None, type=int, choices=[1,0,2,3,4])
    args=parser.parse_args()
    print(args)
    
if __name__=='__main__':
    main()
```

**Comment by chriskiehl** [comment]
This has been fixed in 824e2ff5. It'll be in the 1.0.2 release. 

Thanks for the report! 👍
